__tests__/pet.test.js

```diff
@@ -0,0 +1,281 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+      });
+
+    it('has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toEqual(0);
+    });
+
+    it('has an initial hunger of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.hunger).toEqual(0);
+    });
+  });
+
+describe('growUp', () => {
+
+    it('increase the age by 1', () => {   
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+
+    it('increase the hunger by 5', () => {
```

Comment:
As mentioned in the `pet.js` file, this values could be described by a set of constants in your tests too. Thi makes them less error prone as the same values are used throughout and makes you less likely to slip and use the wrong number